pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -532,6 +532,30 @@ var _ = Describe("Transform", func() {
 				Expect(d.Spec.Strategy.RollingUpdate.MaxUnavailable.IntValue()).To(Equal(0))
 			})
 		})
+
+		Context("for project service configured with pod annotations", func() {
```

Comment:
I guess we can just say `with annotations`? the cases below will explain what's the expected behaviour. The same comment applies to the other specs where we use `pod annotations` in context description.